### ISSUE TITLE
AnnoMI dataset analysis

### ISSUE THREAD

**Issue by rchan26** [ttseriotou/sig-networks#1]
- [x] Client talk type prediction task
  - [x] FFN with current post (tuning on hidden layer sizes, dropout, learning rate)
  - [x] FFN with mean average of history (tuning on same as above, along with window size)
  - [x] FFN with signature/log-signature of history (tuning on same as above, along with dimension reduction technique, dimension to reduce to, signature depth)
  - [x] Fine-tuned BERT for sequence classification
  - [x] BiLSTM on history (tuning on window size, LSTM hidden layer sizes)
  - [x] SWNU network
  - [x] SWNU with BiLSTM network
- [ ] Therapist talk type prediction task
  - [ ] FFN with current post (tuning on hidden layer sizes, dropout, learning rate)
  - [ ] FFN with mean average of history (tuning on same as above, along with window size)
  - [ ] FFN with signature/log-signature of history (tuning on same as above, along with dimension reduction technique, dimension to reduce to, signature depth)
  - [ ] Fine-tuned BERT for sequence classification
  - [ ] BiLSTM on history (tuning on window size, LSTM hidden layer sizes)
  - [ ] SWNU network
  - [ ] SWNU with BiLSTM network
